### DISCUSSION TITLE
Malloy for Windows 64

### DISCUSSION THREAD

**Discussion by free2fork** [malloydata/malloy#540] [General]
Hello everyone.

Currently Malloy isn't available for Windows, any plans to release the support for it ? How soon ?

Cheers!

**Comment by mtoy-googly-moogly** [comment]
Great question!

I wish we had a better answer, but here is the state of Windows Malloy ...

1. We have no objections to Windows.
2. There aren't very many of us, and none of us use Windows.
3. We aren't really "supporting" anything, we are more accurately "making it possible for others to join us in the experiment". 
4. If someone gave us a PR which integrated a windows build we'd be thrilled.
5. Not really sure, if you wait for the Malloy team, when a windows build would show up, "eventually" is the best answer.

**Comment by anikaks** [comment]
I'll just add that it's helpful for us to know how much interest there is in building this or anything else, so if others are reading discussion and wishing "eventually" was "soon," let us know!

**Reply by djouallah** [reply]
probably in the long term, having a web version of Malloy will a more sustainable approach, even PowerBI which is a windows only software is moving slowly to the web.  but today if you have windows, you can't run Malloy and that's sad !!!

**Reply by alexn-s** [reply]
just found malloy and would love a windows build 💖. (djouallah) personally i am not a huge fan of web versions and prefer electron over web

**Reply by lloydtabb** [reply]
@gmaden perhaps a Window version of Composer?  Had another request over the weekend.

**Reply by gmaden** [reply]
Composer is updated and pushed a new release to include a win32-x64 distribution:
https://github.com/malloydata/malloy-composer/releases/latest
